cms/startup.py

```diff
@@ -10,7 +10,9 @@
 
 from openedx.core.lib.django_startup import autostartup
 import django
-from openedx.core.djangoapps.monkey_patch import django_db_models_options
+from openedx.core.djangoapps.monkey_patch import (
```

Comment:
Woops -- came up from the past work. Thought it was intentional so left it alone.